Lib/fontTools/varLib/models.py

```diff
@@ -264,7 +264,17 @@ def upperBound(value, lst):
 			return min(v for v in lst if v > value)
 		else:
 			return value
-
+	def reorderMasters(self, master_list):
+		# Re-order the master item list from the original master font
+		# list order to the sorted location order. This puts the 
+		# default master first, and makes building up the blend data
+		# simpler in some workflows, such as for CFF2 charstrrings.
```

Comment:
This comment doesn't belong here.  Also, I was meaning for `reorderMasters` to take a mapping and reorder to that.  You can then pass `model.reverseMapping` to it.

Reply:
@behdad. Will fix.